package.json

```diff
@@ -24,6 +24,7 @@
     "fullcalendar": "^3.10.0",
     "geodesy": "^1.1.3",
     "history": "^4.7.2",
+    "indico-sui-theme": "github:pferreir/indico-sui-theme#master",
```

Comment:
npm? ;)

Reply:
Well, I was waiting for it to be a little bit more stable.